Content.Server/GameTicking/GameTicker.cs

```diff
@@ -214,7 +215,35 @@ public void EndRound()
 
             RunLevel = GameRunLevel.PostRound;
 
-            SendServerMessage("The round has ended!");
+            //Tell every client the round has ended.
+            var roundEndMessage = _netManager.CreateNetMessage<MsgRoundEndMessage>();
+            roundEndMessage.GamemodeTitle = MakeGamePreset().ModeTitle;
+
+            //Get the timespan of the round.
+            var gameTime = IoCManager.Resolve<IGameTiming>();
+            roundEndMessage.RoundDuration = gameTime.RealTime;
```

Comment:
`RealTime` is not the length of the current round, it's server uptime. You're gonna need to store the time inside `StartRound()` and refer to that later instead.